ci/jjb/jobs/projects.yaml

```diff
@@ -57,8 +59,10 @@
     name: pulp-dev-2-17
     build_version:
     - 2-master:
-        pulp_version: '2.17'
+        pulp_version: '2.18'
```

Comment:
@Ichimonji10 i am unsure about this line?

Reply:
Yes, I think this is correct.